validation/build.gradle

```diff
@@ -15,7 +15,9 @@ dependencies {
     api project(":core-reactive")
     api libs.managed.validation
 
-    compileOnly libs.managed.gorm
+    compileOnly(libs.managed.gorm) {
+        exclude(module: 'groovy')
```

Comment:
mmm, we should figure out why an exclude is needed here

Reply:
`libs.managed.gorm` is `org.grails:grails-datastore-core:7.3.2` at time of writing which has a [compile dependency on Groovy 3](https://search.maven.org/artifact/org.grails/grails-datastore-core/7.3.2/jar)

```
    <dependency>
      <groupId>org.codehaus.groovy</groupId>
      <artifactId>groovy</artifactId>
      <version>3.0.11</version>
      <scope>compile</scope>
    </dependency>
```

This is just a compile only dependency, but we may be breaking gorm integration until grals release a Groovy 4 version of their libraries that we consume